osprey_worker/src/osprey/engine/stdlib/udfs/tests/test_strings.py

```diff
@@ -182,17 +182,17 @@ class Scenario:
         Scenario('🔞t3xt'),
         Scenario('ΠOPQ 100-', ['nop', '100']),
         Scenario('Te.xt'),
-        Scenario('∏opq'),
+        Scenario('∏opq', 'nopq'),
```

Comment:
What does the second param do?

Reply:
yeah these are a bit tricky to understand without looking at how `Scenario` is consumed. But essentially, it'll use the default value of `text` if this param is missing, otherwise it'll use this param's value to compare against the formatted string.